CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Changes Since Last Release
 
-### Changed defaults / behaviours
+## 4.0.2 \[2023-11-16\]
```

Comment:
I might be misremembering this but... isn't it the "Changes Since Last Release" that should be changed to the version number, and the subdivision into "Changed defaults / behavio(u)rs" (😉) etc. should remain intact...?

Reply:
Yep, you are right 🤦